gui/default/syncthing/core/syncthingController.js

```diff
@@ -1072,8 +1074,11 @@ angular.module('syncthing.core')
 
         $scope.deviceStatus = function (deviceCfg) {
             var status = '';
-
             if ($scope.deviceFolders(deviceCfg).length === 0) {
+                var unused = true;
+            }
+
+            if (unused) {
```

Comment:
That might be correct JS, but scoping at least looks wrong - something like this I'd expect

```
            var unused =($scope.deviceFolders(deviceCfg).length === 0

            if (unused) {
```

Reply:
Done. Is there any difference in scope though? Variables declared with `var` are available inside the whole function anyway.

Reply:
That's what I meant: It likely is correct and doesn't matter, but looks wrong and definitely isn't elegant (to exagerate you never want to do `if condition { conditionVariable = true } else { conditionVariable = false}` when you can just do `conditionVariable = condition`.